dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -737,7 +749,8 @@ internal constructor(
             viewTrackingStrategy = ActivityViewTrackingStrategy(false),
             longTaskTrackingStrategy = MainLooperLongTaskStrategy(DEFAULT_LONG_TASK_THRESHOLD_MS),
             rumEventMapper = NoOpEventMapper(),
-            backgroundEventTracking = false
+            backgroundEventTracking = false,
+            vitalsMonitorUpdateFrequency = VitalsUpdateFrequency.AVERAGE
```

Comment:
so we are moving from 100 ms to 500 ms by default?

Reply:
Yeah, that was the agreement :)